descriptors/ModuleDescriptor-template.json

```
@@ -77,6 +77,42 @@
           "permissionsRequired": [
             "specification-storage.fields.item.delete"
           ]
+        },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/specification-storage/fields/{fieldId}/indicators",
+          "permissionsRequired": [
+            "specification-storage.specification.indicators.collection.get"
```

```suggestion
            "specification-storage.field.indicators.collection.get"
```
Maybe in such format? To fit path pattern?